project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/envision "0.1.0-SNAPSHOT"
+(defproject clojurewerkz/envision "0.1.1-SNAPSHOT"
```

Comment:
Never change version in a pull request. It's up to the maintainer to decide when to do that and what the new version should be.